From d78ec3f29eca74d17a01d717e10ef6fad1a85a15 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 17 Feb 2014 08:20:41 +0000
Subject: [PATCH] run the gunlib checks with same compiler as when building

The gnulib will be built with $CC and not $CXX so we should not run the
gnulib tests with $CXX.

This fixes lavv17/lftp#59
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index faca8e5..5d52f1a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -237,9 +237,7 @@ fi
 AM_PROG_LIBTOOL
 LIBTOOL="$LIBTOOL --silent"
 
-AC_LANG_PUSH(C++)
 gl_INIT
-AC_LANG_POP(C++)
 
 if test x$ac_cv_func_wcwidth != xyes || test x$REPLACE_WCWIDTH = x1; then
    gl_LIBOBJ([wcwidth])
-- 
1.8.5.4

